src/lib/assetBridger/ethBridger.ts

```diff
@@ -194,6 +201,37 @@ export class EthBridger extends AssetBridger<
     }
   }
 
+  /**
+   * Deposit ETH from L1 onto a different L2 address
+   * @param params
+   * @returns
+   */
+  public async depositTo(
```

Comment:
Can we make this method signature be more closely in line with [Erc20Bridger.deposit](https://github.com/OffchainLabs/arbitrum-sdk/blob/main/src/lib/assetBridger/erc20Bridger.ts#L90), so it has `l2Provider` and `retryableGasOverrides` as part of the `params` object?

Reply:
Can we also add support for specifying a transaction request instead of params?

Reply:
Yes to both. I'll add my thoughts:
- I chose to have the signature be closer to EthBridger.deposit instead of Erc20.deposit (I figured it would make sense being in the same file), but it's true that with all the extra parameters it ends up being quite different anyway. I'll change it.
- About the support for specifying a transaction request, I wanted to avoid calling `getDepositRequest` to standarize the request object (because that's already done by `createRetryableTicket`), but I think I can come up with a way to support it and keep the function simple.

Reply:
We might need the `l2Provider` as a separate parameter. We need it for `L1ToL2MessageCreator.createRetryableTicket()` and is not part of the regular transaction request type (`L1ToL2TxReqAndSigner`, `L1ToL2TxReqAndSignerProvider `and others).
As an alternative, I can also create a new type for the transaction request, extending from an existing one and add that property.
I'm leaning towards adding it as a separate parameter on the function, but I can go either way. What do you think?

(We shouldn't have that problem with GasOverrides as `L1ToL2TransactionRequest` includes those properties)

Reply:
https://github.com/OffchainLabs/arbitrum-sdk/pull/238/files#diff-a977cd005aca51be6f05bc7e1c7c1bf6d734b62b2c45c84b05e2eb0c3c3c6fffR233
I ended up adding the `l2Provider` to the `L1ToL2TxReqAndSigner` type in the function signature, so we would only need one parameter for the method. What do you think?